server/src/main/java/com/genymobile/scrcpy/FakeContext.java

```diff
@@ -23,11 +24,24 @@ private FakeContext() {
 
     @Override
     public String getPackageName() {
+        if (Os.getuid() == 1000) {
```

Comment:
I prefer to call `getPackageNameStatic` here and in `getOpPackageName` to not duplicate code.